tools/photofinish/.photofinish.toml

```diff
@@ -0,0 +1,12 @@
+[first-test-scenario]
+files = [
+  "../../test/fixtures/discovery/host/expected_published_host_discovery.json",
+  "../../test/fixtures/discovery/sap_system/sap_system_discovery_application.json",
```

Comment:
Ah this is smart, when you told me about the toml file i imagined a super complicated stuff with the nodes definition which actually we don't really need. I like this